scripts/e2e-widget.js

```diff
@@ -15,6 +17,7 @@ const fileNames = [
   "responsibleaidashboard-housing-decision-making",
   "responsibleaidashboard-multiclass-dnn-model-debugging"
 ];
+const flightCombinations = [[], ["newModelOverviewExperience"]];

```

Comment:
why do we have this empty array?

Reply:
The default is no flights. I'll add a comment to document this!